xarray/core/dataarray.py

```diff
@@ -1128,11 +1129,11 @@ def from_dict(cls, d):
 
         d = {'dims': ('t'), 'data': x}
 
-        d = {'coords': {'t': {'dims': 't', 'data': t,
-                              'attrs': {'units':'s'}}},
-             'attrs': {'title': 'air temperature'},
-             'dims': 't',
-             'data': x,
+        d = {'coords': {'t': {'dims': 't', 'data': t, \
```

Comment:
The `\n` and the like are a little bad in the docstring because they aren't valid Python code.